deltachat-rpc-client/README.md

```diff
@@ -0,0 +1,33 @@
+# Delta Chat RPC python client
+
+RPC client connects to standalone Delta Chat RPC server `deltachat-rpc-server`
+and provides asynchronous interface to it.
+
+## Getting started
+
+To use Delta Chat RPC client, first build a `deltachat-rpc-server` with `cargo build -p deltachat-rpc-server`.
+Install it anywhere in your `PATH`.
+
+## Testing
+
+1. Build `deltachat-rpc-server` with `cargo build -p deltachat-rpc-server`.
+2. Run `tox`.
```

Comment:
was trying this and got error, because it must be run inside the `deltachat-rpc-client` folder (I guess) so would be nice to add an step with `cd deltachat-rpc-client` or something like that for the dummy :D